twistr/events.go

```diff
@@ -532,6 +568,10 @@ func PlayCubanMissileCrisis(s *State, player Aff) {
 	   opponent will lose the game. This card’s Event may be canceled, at any time,
 	   if the USSR removes 2 Influence from Cuba or the US removes 2 Influence from
 	   West Germany or Turkey. */
+	s.Defcon = 2
+	// XXX: Shit, does player get this, or opposition? Also, implement thermonuclear war
+	// in the coup code
+	s.TurnEvents[CubanMissileCrisis] = player
```

Comment:
I agree with assigning it to player. That's most consistent with what we've done so far.
It will be more evident from reading, in the coup code, if s.Effect(CubanMissileCrisis, player.Opp()).